src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final int INITIAL_SUM = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int sumBuy = readFromFile(fromFileName, BUY);
+        int sumSupply = readFromFile(fromFileName, SUPPLY);
```

Comment:
it should be 3 separate methods
each of responsible only for one particular task (logic) 
it's Single responsibility principle 
```suggestion
        int[] output = readFromFile(fromFileName);
        String result = getReport(output);
        writeToFile(result, toFileName);
```

Reply:
@Elena-Bruyako 
I think it's a bad solution to solve the problem this way. You suggest dividing the logic, but this will only add a few unnecessary actions like creating additional arrays and going through them, instead of immediately reading the information from the file and passing it to separate variables. I think the best solution would be to rename the method to getSumByKeyword, this way we don't do unnecessary operations, but immediately work directly with the file. Taking content and processing it several times is bad practice in my opinion.

Or we can create a getData method, not "readFromFile", because if we read, the method should not return anything, and then what's the point of it. Then this method (getData) collects them and passes them to a separately created "getSum", which we pass to the corresponding variables. But even so, this will be the wrong solution because we create unnecessary operations and arrays.

Reply:
Уяви ситуацію, коли в тебе буде не дві операції. Для кожної з них ти будеш викликати один і той самий метод. Краще зробити певну логіку в методі, за яку він відповідає та передати результат